cognite_toolkit/_cdf_tk/utils/useful_types.py

```diff
@@ -8,10 +8,11 @@
 JsonVal: TypeAlias = None | str | int | float | bool | dict[str, "JsonVal"] | list["JsonVal"]
 
 AssetCentricDestinationType: TypeAlias = Literal["assets", "files", "events", "timeseries", "sequences"]
-AssetCentricType: TypeAlias = Literal["asset", "file", "event", "timeseries", "sequence", "fileAnnotation"]
+AssetCentricType: TypeAlias = Literal["asset", "file", "event", "timeseries", "sequence"]
+AssetCentricTypeExtended: TypeAlias = Literal["asset", "file", "event", "timeseries", "sequence", "annotation"]
 AssetCentricResource: TypeAlias = Asset | FileMetadata | Event | TimeSeries
 AssetCentricResourceExtended: TypeAlias = Asset | FileMetadata | Event | TimeSeries | Annotation
-AssetCentricKind: TypeAlias = Literal["Assets", "Events", "TimeSeries", "FileMetadata", "FileAnnotations"]
+AssetCentricKind: TypeAlias = Literal["Assets", "Events", "TimeSeries", "FileMetadata"]
```

Comment:
Using extended to mark whether annotations are part of it or not, this is because in migration we need the extended version, while for example hierarchy download we use the non-extended version.

Furthermore, no more difference between annotation types.